apps/src/codebridge/Workspace/index.tsx

```diff
@@ -53,7 +54,7 @@ const Workspace = () => {
     <PanelContainer
       id="editor-workspace"
       headerContent={headerContent}
-      rightHeaderContent={<HeaderButtons />}
+      rightHeaderContent={<HeaderButtons skipUrl={skipUrl} />}
```

Comment:
Can we get the `skipUrl` from redux in `HeaderButtons` rather than passing it as a prop?

Reply:
Sure - will update.